### PR TITLE
include crafting mats in consumables counter/update seaonal mats

### PR DESCRIPTION
**Pull request by nev-r**
image

### CODE REVIEW

**Comment by nev-r** [comment]
need to re-do this with profile strings, some numbers are misreported by the api

File: src/app/store-stats/MaterialCounts.tsx [hunk 1]
```diff
@@ -11,10 +15,15 @@ import styles from './MaterialCounts.m.scss';
 const retiredMats = [31293053, 49145143, 1305274547, 2014411539];
 const showMats = spiderMats.filter((m) => !retiredMats.includes(m));
 const goodMats = [2979281381, 4257549984, 3853748946, 4257549985, 3702027555];
-const seasonal = [1425776985, 1776857076];
-
+const seasonal = [747321467];
```
Comment (robojumper) [inline src/app/store-stats/MaterialCounts.tsx:18]:
```suggestion
const seasonal = [747321467, 178490507];
```
include Risen Umbral Energy, the currency used to focus engrams, here?

Comment (nev-r) [reply src/app/store-stats/MaterialCounts.tsx:18]: thanks

**Comment by robojumper** [comment]
Any particular reason this uses the objectives and not the dummy items for resonant materials? e.g. https://data.destinysets.com/i/InventoryItem:163842161

**Comment by nev-r** [comment]
mmm leftover choice from when the data was objective-driven. you're right i''ll swap them out

**Review by robojumper — APPROVED** [review]

**Pull request merged**